app/views/claim_mailer/submitted.text.erb

```diff
@@ -1,4 +1,4 @@
-Dear <%= @claim.full_name %>,
+Dear <%= @claim.first_name %>,
```

Comment:
Do we want to address them as their full name here? It might be good to have a `full_name` method that pieces the name parts together